vcr/fixtures/cassette/TestAllInSeries_fixtures_computenodetemplate.yaml

```diff
@@ -0,0 +1,543 @@
+                  }
+                ]
+              }
+            }
+        headers:
+            Cache-Control:
+                - private
+            Content-Type:
+                - application/json; charset=UTF-8
+            Date:
+                - Tue, 12 Mar 2024 00:22:23 GMT
+            Server:
+                - ESF
+            Vary:
+                - Origin
+                - X-Origin
+                - Referer
+            X-Content-Type-Options:
+                - nosniff
+            X-Frame-Options:
+                - SAMEORIGIN
+            X-Xss-Protection:
+                - "0"
+        status: 404 Not Found
+        code: 404
+        duration: 198.036199ms
+    - id: 1
+      request:
+        proto: HTTP/1.1
+        proto_major: 1
+        proto_minor: 1
+        content_length: 417
+        transfer_encoding: []
+        trailer: {}
+        host: compute.googleapis.com
+        remote_addr: ""
+        request_uri: ""
+        body: |
+            {"cpuOvercommitType":"NONE","description":"Node template for sole tenant nodes running in us-central1, with 96vCPUs and any amount of memory on any machine type.","name":"computenodetemplate-le4q3bkhysnhydb2zckq","nodeAffinityLabels":{"cnrm-test":"true","managed-by-cnrm":"true","memory_guarantee":"false"},"nodeTypeFlexibility":{"cpus":"96","memory":"any"},"region":"projects/cnrm-yuhou/global/regions/us-central1"}
+        form: {}
+        headers:
+            Content-Type:
+                - application/json
+            User-Agent:
+                - Terraform/ (+https://www.terraform.io) Terraform-Plugin-SDK/2.10.1 terraform-provider-google-beta/dev
+        url: https://compute.googleapis.com/compute/beta/projects/cnrm-yuhou/regions/us-central1/nodeTemplates?alt=json
+        method: POST
+      response:
+        proto: HTTP/2.0
+        proto_major: 2
+        proto_minor: 0
+        transfer_encoding: []
+        trailer: {}
+        content_length: -1
+        uncompressed: true
+        body: |
+            {
+              "kind": "compute#operation",
+              "id": "4057836747365686959",
+              "name": "operation-1710202944238-6136ba6fc31ca-90ae9715-222761eb",
+              "operationType": "compute.nodeTemplates.insert",
+              "targetLink": "https://www.googleapis.com/compute/beta/projects/cnrm-yuhou/regions/us-central1/nodeTemplates/computenodetemplate-le4q3bkhysnhydb2zckq",
+              "targetId": "538944551121985199",
+              "status": "RUNNING",
+              "user": "yuhou@google.com",
```

Comment:
Nit: should probably replace your username also

Reply:
Addressed in PR #1349